ranger/data/scope.sh

```diff
@@ -249,17 +238,22 @@ handle_image() {
     # }
 
     # case "${FILE_EXTENSION_LOWER}" in
-    #     ## 3D models
-    #     ## OpenSCAD only supports png image output, and ${IMAGE_CACHE_PATH}
-    #     ## is hardcoded as jpeg. So we make a tempfile.png and just
-    #     ## move/rename it to jpg. This works because image libraries are
-    #     ## smart enough to handle it.
-    #     csg|scad)
-    #         openscad_image "${FILE_PATH}" && exit 6
-    #         ;;
-    #     3mf|amf|dxf|off|stl)
-    #         openscad_image <(echo "import(\"${FILE_PATH}\");") && exit 6
-    #         ;;
+        ## 3D models
```

Comment:
I'm not sure the comment style should be changed, it clearly indicates that you need to uncomment the case statement rather than just the section you want to apply.

Reply:
I changed it because I thought this style better indicates that you can just uncomment the case statement and one of the sections without uncommenting the other.

Reply:
Let's stick with the status quo.